### PR TITLE
Update French translation

### PR DESCRIPTION
**Pull request by NathanBnm**
I just updated French translation :smile:

**This PR is ready for review**

### CODE REVIEW

File: po/extra/fr.po [hunk 1]
```diff
@@ -44,45 +44,97 @@ msgid ""
 msgstr ""
```
Comment (ryonakano) [inline po/extra/fr.po]: I don't expect all source strings are always translated so I just comment to ask, but is it just you forgot to translate or you have some difficulties to translate this string (or another reason) that this line is not being translated?

File: po/extra/fr.po [hunk 2]
```diff
@@ -44,45 +44,97 @@ msgid ""
...
 #: data/com.github.ryonakano.reco.appdata.xml.in:18
 msgid ""
 "Autosaving or always-ask-where-to-save workflow. You can select whether the "
 "app saves recordings automatically or you choose a destination each time on "
 "saving them"
 msgstr ""
+"Comportement de sauvegarde automatique ou demande d'emplacement de "
+"destination pour chaque enregistrement. Vous pouvez sélectionner si "
+"l'application sauvegarde automatiquement les enregistrements ou si vous "
+"préferez sélectionner un emplacement de destination à chaque sauvegarde."
```
Comment (ryonakano) [inline po/extra/fr.po]: The translated string seems to have a period which the source string does not have at the end, but is it intentional?

File: po/extra/fr.po (hunk 1 above)
Comment (NathanBnm) [reply po/extra/fr.po]: You're right I just forgot this one. I'll translate it. Thanks for the report :smile:!

File: po/extra/fr.po (hunk 2 above)
Comment (NathanBnm) [reply po/extra/fr.po]: It's not intentional, I'll remove it.

**Comment by NathanBnm** [comment]
@ryonakano thanks for your comments. It should be good now!

**Pull request merged**